test/local/log-stats.js

```diff
@@ -10,11 +10,16 @@ var config = require('../../config')
 config.logLevel = 'info'
 config.statInterval = 100
 
-var log = require('../../log')(config.logLevel, 'db-api')
+var log = require('../../logging')('test.local.log-stats')
 
-// monkeypatch log.stat to hook into db/mysql.js:statInterval
+// monkeypatch log.info to hook into db/mysql.js:statInterval
 var dfd = P.defer()
-log.stat = function(stats) {
+var consumeFirstN = 1
+log.info = function(msg, stats) {
```

Comment:
Yes, for this test only, which is doing a check that the stats are being output.

The original test was on `.trace()` therefore could check this on the first trace output, however with using `.info()` and there being an info call prior to this one, I had to gobble up the first output and essentially read the second. I think it's ok for this test only.